dragonfly/block/grass_plant.go

```diff
@@ -0,0 +1,157 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/dragonfly/block/grass"
+	"github.com/df-mc/dragonfly/dragonfly/item"
+	"github.com/df-mc/dragonfly/dragonfly/item/tool"
+	"github.com/df-mc/dragonfly/dragonfly/world"
+	"github.com/go-gl/mathgl/mgl64"
+	"math/rand"
+)
+
+// GrassPlant is a transparent plant block which can be used to obtain seeds and as decoration.
+type GrassPlant struct {
+	noNBT
+	transparent
+	empty
+
+	UpperBit bool
+
+	grass.Grass
```

Comment:
Not sure if this should be embedded. Might be better to just make it a normal field (`Grass` or `GrassType`).

Reply:
Renamed to Type.